### PR TITLE
Another update to glossary :)

### PR DESCRIPTION
**Pull request by AkiNikolaidis**
Closes # .

Changes proposed in this pull request:

-
-

### CODE REVIEW

**Review by htwangtw — COMMENTED** [review]
Would it be possible to update your other PR? 
On your own fork, do a pull request of `patch-2` to `patch-1`, this way we can have all your changes in the PR associated to `patch-1`.

Happy to chat a bit late!

File: brainhack_book/glossary.md [hunk 1]
```diff
@@ -344,7 +347,7 @@ Reporting bias occurs when certain aspects of a study are systematically not rep
...
+Reproducibility is a spectrum and instructors should choose the definition most used by their audience. Generally speaking, reproducible research makes it possible to obtain similar results of a study or experiment and independent results obtained with the same methods but under different conditions (i.e., pertains to results). Some break the definition into levels of reproducibility, including computationally reproducible (also called "reproducible"): where code and data can be analyzed in a similar manner as in the original research to achieve the same results, and empirically reproducible (also called “replicable”): where an independent researcher can repeat a study using the same methods but creating new data. Generally speaking, when discussing reproducibility it is important to specify the changing conditions across which measurement/findings are being reproduced. Computational reproducibility, cross-sample reproducibility, and cross-site reproducibility and other forms all specify the changing conditions under which reproducibility is being measured.
```
[the remark this answers is not in the document]
Comment (AkiNikolaidis) [reply brainhack_book/glossary.md:350]: > Would it be possible to update your other PR?
> On your own fork, do a pull request of `patch-2` to `patch-1`, this way we can have all your changes in the PR associated to `patch-1`.
> 
> Happy to chat a bit late!

Yes! Happy to do that- unsure how though- lmk if you can chat quickly about it  at some point

[the remark this answers is not in the document]
Comment (AkiNikolaidis) [reply brainhack_book/glossary.md:350]: The PR from patch 1 to patch 2 i mean

**Comment by AkiNikolaidis** [comment]
Trying to merge it into one. Lmk if this works

**Pull request closed without merging**